doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -717,6 +717,9 @@ Modem libraries
     * The ``CONFIG_LTE_NETWORK_USE_FALLBACK`` Kconfig option.
       Use the :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT` or :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_NBIOT_GPS` Kconfig option instead.
       In addition, you can control the priority between LTE-M and NB-IoT using the :kconfig:option:`CONFIG_LTE_MODE_PREFERENCE` Kconfig option.
+    * The :c:func:`lte_lc_reduced_mobility_get` and :c:func:`lte_lc_reduced_mobility_set` functions have been deprecated.
+      Please refer to the :ref:`migration guide <migration_2.8>` for more details.
+    * The :c:enum:`lte_lc_reduced_mobility_mode` type has been deprecated. Please refer to the :ref:`migration guide <migration_2.8>` for more details.
```

Comment:
```suggestion
    * The :c:func:`lte_lc_reduced_mobility_get` and :c:func:`lte_lc_reduced_mobility_set` functions have been deprecated.
      Refer to the :ref:`migration guide <migration_2.8>` for more details.
    * The :c:enum:`lte_lc_reduced_mobility_mode` type has been deprecated. 
      Refer to the :ref:`migration guide <migration_2.8>` for more details.
```
Could move these points to updated, since these are deprecated, and not yet removed. For similar example : https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-2.6.0.html#modem_libraries